data/docs_markdown/Rewards_&_Wallets.md

```
@@ -0,0 +1,81 @@
+# Rewards & Wallets
+This document explains the financial aspects of supplying compute power to the IO network.
+### [](https://docs.io.net/docs/rewards-wallets#table-of-contents)
+  * [Rewards and Fees](https://docs.io.net/docs/rewards-wallets#rewards-and-fees)
+  * [Add Wallet to Account](https://docs.io.net/docs/rewards-wallets#add-wallet-to-account)
+  * [Withdraw Funds from Your Account](https://docs.io.net/docs/rewards-wallets#withdral-funds-from-your-account)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)